.github/workflows/ci.yml

```diff
@@ -132,6 +132,8 @@ jobs:
             DOCKERIMAGE: condaforge/linux-anvil-aarch64
             MINIFORGE_NAME: "Mambaforge-pypy3"
             OS_NAME: "Linux"
+            # Reduce the test matrix because the builds timeouts on emulated architectures
+            TEST_IMAGE_NAMES: "ubuntu:22.04 ubuntu:16.04 centos:7"
```

Comment:
Let's keep only one. The one where the timeout happened barely ran two tests. https://pipelines.actions.githubusercontent.com/serviceHosts/e400fe88-afca-422a-abd9-23e00445dee4/_apis/pipelines/1/runs/6160/signedlogcontent/16?urlExpires=2022-10-23T19%3A25%3A43.9598492Z&urlSigningMethod=HMACV1&urlSignature=WWLz9fSOwtV%2F6x%2BVpSGa47kLLBUt7FOlRi7Z%2FOxGBaM%3D

Reply:
K. one has a modern ubuntu, the other has centos 7. I feel like that is a good spread for these.